.github/workflows/publish-harness.yaml

```diff
@@ -96,29 +96,27 @@ jobs:
     runs-on: ubuntu-20.04
     needs: [publish-harness-base]
     env:
-      TERRAFORM_VERSION: 1.8.2
+      TOFU_VERESION: 1.7.2
     strategy:
       matrix:
         versions:
-        - full: 1.8.2
-          tag: 1.8.2
-        - full: 1.8.2
+        - tofu: 1.7.2
           tag: "1.8"
-        - full: 1.7.5
+        - tofu: 1.7.1
           tag: '1.7'
-        - full: 1.6.6
+        - tofu: 1.6.2
           tag: '1.6'
-        - full: 1.5.7
+        - tofu: 1.6.2
           tag: '1.5'
-        - full: 1.4.7
+        - tofu: 1.6.2
           tag: '1.4'
-        - full: 1.3.10
+        - tofu: 1.6.2
           tag: '1.3'
-        - full: 1.2.9
+        - tofu: 1.6.2
           tag: '1.2'
-        - full: 1.1.9
+        - tofu: 1.6.2
           tag: '1.1'
-        - full: 1.0.11
+        - tofu: 1.6.2
```

Comment:
are these versions actually aligned?  Looks like the tofu version is always `1.6.2`?

Reply:
yes, i double checked the [OpenTofu Migration Docs](https://opentofu.org/docs/intro/migration/terraform-1.5-or-lower/)
> When migrating from Terraform 1.5.x or lower, please migrate to OpenTofu 1.6.2 first